packages/core/src/entity/Collection.ts

```diff
@@ -57,23 +57,29 @@ export class Collection<T extends object, O extends object = object> extends Arr
    * Gets the count of collection items from database instead of counting loaded items.
    * The value is cached, use `refresh = true` to force reload it.
    */
-  async loadCount(refresh = false): Promise<number> {
-    if (!refresh && Utils.isDefined(this._count)) {
+  async loadCount(refresh = false, options: LoadCountOptions<T> = {}): Promise<number> {
```

Comment:
i meant something like this:
```suggestion
  async loadCount(options: LoadCountOptions<T> | boolean = {}): Promise<number> {
    options = typeof options === 'boolean' ? { refresh: options } : options;
```

Reply:
and i will remove the boolean variant in v6 as it will be a breaking change